src/components/Details/DetailObject.tsx

```diff
@@ -1,10 +1,13 @@
 import * as React from 'react';
 import Label from '../Label/Label';
+import { Icon } from 'patternfly-react';
 
 interface DetailObjectProps {
   name: string;
   detail: any;
   labels?: string[];
+  exclude?: string[];
+  validation?: any;
```

Comment:
Perhaps validation should have a type here.
I see there is at least a .icon and a .message, so, I guess it would be nice to have a strong type here if possible.